### PR TITLE
Feature flags

### PR DESCRIPTION
**Pull request by Ciuffi**
I've added a page for editing Feature Flags.

Here is the page being viewed:
image
Here is the page being edited:
image

### CODE REVIEW

**Review by ianmah — COMMENTED** [review]
Maybe to keep the UI consistent we could remove the border-radius and border?

File: components/page.js [hunk 1]
```diff
@@ -116,6 +116,22 @@ export default ({ hackathons, currentPath, children }) => {
...
+            <NavItem
+              onClick={() => {
+                if (!window.location.href.includes('FeatureFlags')) {
+                  setTimeOut(
+                    setTimeout(() => {
+                      setLoading(true);
+                    }, 750)
+                  );
+                }
+              }}
```
Comment (ianmah) [inline components/page.js:129]: Maybe we could reduce this to a helper?

Comment (Ciuffi) [reply components/page.js:129]: hm maybe this change is better in your Link PR? we can merge that one in first and I'll copy the helper for this branch.

**Comment by Ciuffi** [comment]
> Maybe to keep the UI consistent we could remove the border-radius and border?

I think we should remove the border radius but I think it looks a little weird without at least a small border. 

Without the border its just a slightly changed background with no edge with the card background.

File: components/page.js (hunk 1 above)
Comment (ianmah) [reply components/page.js:129]: soundsgoodtome

**Comment by Ciuffi** [comment]
Preview link ferda https://pr-33.d1shnh93w9du7n.amplifyapp.com/landing

**Comment by Ciuffi** [comment]
@ianmah integrated your helper function

**Review by ianmah — APPROVED** [review]

**Review by andykwan1124 — APPROVED** [review]

**Pull request merged**